fastlane/lanes/release.rb

```diff
@@ -13,11 +13,58 @@
   #
   desc 'Executes the initial steps needed during code freeze'
   lane :code_freeze do |options|
+    # Verify that there's nothing in progress in the working copy
+    ensure_git_status_clean
+
+    # Check out the up-to-date default branch, the designated starting point for the code freeze
+    Fastlane::Helper::GitHelper.checkout_and_pull(DEFAULT_BRANCH)
+
+    # Make sure that Gutenberg is configured as expected for a successful code freeze
     gutenberg_dep_check
-    ios_codefreeze_prechecks(options)
 
-    ios_bump_version_release
-    new_version = get_app_version
+    # The `release_version_next` is used as the `new internal release version` value because the external and internal
+    # release versions are always the same.
+    message = <<-MESSAGE
+      Code Freeze:
+      • New release branch from #{DEFAULT_BRANCH}: release/#{release_version_next}
+
+      • Current release version and build code: #{release_version_current} (#{build_code_current}).
+      • New release version and build code: #{release_version_next} (#{build_code_code_freeze}).
+
+      • Current internal release version and build code: #{release_version_current_internal} (#{build_code_current_internal})
+      • New internal release version and build code: #{release_version_next} (#{build_code_code_freeze_internal})
+    MESSAGE
+
+    UI.important(message)
+    UI.user_error!('Aborted by user request') if !options[:skip_confirm] && !UI.confirm('Do you want to continue?')
+
+    # Create the release branch
+    UI.message 'Creating release branch...'
+    ensure_git_branch(branch: DEFAULT_BRANCH)
+    Fastlane::Helper::GitHelper.create_branch("release/#{release_version_next}", from: DEFAULT_BRANCH)
```

Comment:
Should we remove the `ensure_git_branch` given we already checked out `DEFAULT_BRANCH` _and_ the `create_branch` method has `from: DEFAULT_BRANCH`?

Reply:
Agreed! Removed in https://github.com/wordpress-mobile/WordPress-iOS/commit/a2f00c22e3f48159b23ffb66112be825c7ecedb9